nss-cohort-36/bangazon-api-virtual_magicians
# updated Product fixture with images

# Description
You can now see the images on Product List and Product Details view. **Make sure to load new Products fixture.**
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-product-image`
4. `python manage.py loaddata product`
5. `python manage.py runserver`
6. Switch to the React pr

# Checklist:
